runtime/compiler/z/codegen/S390J9CallSnippet.cpp

```diff
@@ -233,7 +233,11 @@ TR::S390J9CallSnippet::generateInvokeExactJ2IThunk(TR::Node * callNode, int32_t
       cursor += 2;
       }
 
+#if defined(JITSERVER_SUPPORT)
+   *(uintptrj_t *) cursor = (uintptrj_t) cg->fej9()->getInvokeExactThunkHelperAddress(comp, dispatcherSymbol, callNode->getDataType());
```

Comment:
Leaving just the if part of the ifdef the code is equivalent even for non-jitaas